dist/s_bazel.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import pprint
+
+platform2files = {}
+with open("filelist") as f:
+    for a in [l.split() for l in f if l != "\n" and not l.startswith("#")]:
+        file, platform = a[0], f"WT_FILELIST_{a[1]}" if len(a) > 1 else "WT_FILELIST"
+        platform2files.setdefault(platform, [])
+        platform2files[platform].append(file)
+
+res = """
+# DO NOT EDIT: automatically built by dist/s_bazel.
+
+# This file is only used by external projects building WiredTiger via Bazel.
+"""
+for platform, files in platform2files.items():
+    res += f"\n{platform} = " + pprint.pformat(files) + "\n"
+
+try:
+    with open("filelist.bzl") as f:
+        if res == f.read():
+            exit()
+except OSError:
+    pass
```

Comment:
Should we tell `s_all` there is an issue and fail?

Reply:
It's not an issue - the file didn't exist and we create it.
If file didn't exist or changed, it will be reported and s_all will treat it as error.